libglusterfs/src/glusterfs/mem-pool.h

```diff
@@ -158,6 +158,18 @@ __gf_default_realloc(void *oldptr, size_t size)
 
 #define GF_FREE(free_ptr) __gf_free(free_ptr)
 
+#if defined(GF_DISABLE_MEMPOOL)
+#define mem_get(mem_pool)                                                      \
+    GF_MALLOC((unsigned long)mem_pool, gf_common_mt_mem_pool);
+#define mem_get0(mem_pool)                                                     \
+    GF_CALLOC(1, (unsigned long)mem_pool, gf_common_mt_mem_pool);
+#define mem_put(mem_pool) GF_FREE(mem_pool);
```

Comment:
Don't use `mem_pool` as the argument name. This macro is not receiving a memory pool object, it's receiving a pointer. Using this name is misleading for code readers.

Reply:
done